modules/face/samples/sampleKazemi.cpp

```diff
@@ -0,0 +1,113 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include "opencv2/face/face_alignment.hpp"
+#include "opencv2/core.hpp"
+#include "opencv2/highgui.hpp"
+#include "opencv2/imgproc.hpp"
+#include "opencv2/imgcodecs.hpp"
+
+using namespace std;
+using namespace cv;
+using namespace cv::face;
+
+int main(int argc,char** argv){
+    //Give the path to the directory containing all the files containing data
+    if(argc<2){
```

Comment:
This works, but it would be better to process the input data with the command line parser. It seems like overkill, but it is universal and allows user to build more complex input argument structures. An example: https://github.com/opencv/opencv/blob/master/apps/annotation/opencv_annotation.cpp#L223